src/web/components/SignInGate/SignInGateSelector.tsx

```diff
@@ -74,13 +75,13 @@ const tests: ReadonlyArray<ABTest> = [
 const testVariantToGateMapping: GateTestMap = {
     'patientia-control-1': gatePatientiaControl,
     'patientia-variant-1': gatePatientiaVariant,
-    'main-control-1': gateMainControl,
-    'main-variant-1': gateMainVariant,
+    'main-control-2': gateMainControl,
```

Comment:
@coldlink is it correct to bump the control/variant Ids for this variation?

Reply:
yeah thats right